### PR TITLE
[ConnectedVMware] Added test cases

### PR DESCRIPTION
**Pull request by BethanyZhou**
## Checklist

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/ConnectedVMware/README.md [hunk 1]
```diff
@@ -62,7 +62,8 @@ directive:
     remove: true
   # Remove the set-* cmdlet
   - where:
-      verb: Set
```
Comment (BethanyZhou) [inline src/ConnectedVMware/README.md:65]: Why you want to keep set-* cmdlets?

File: src/ConnectedVMware/test/AzConnectedVMwareDatastore.Tests.ps1 [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  }
+  . ($mockingPath | Select-Object -First 1).FullName
+}
+
+Describe 'AzConnectedVMwareDatastore' {
+    It 'CreateExpanded' {
+        New-AzConnectedVMwareDatastore -Name $env.datastoreName -ResourceGroupName $env.resourceGroupName -Location $env.location -ExtendedLocationName $env.extendedLocationName -ExtendedLocationType $env.extendedLocationType -InventoryItemId "/subscriptions/204898ee-cd13-4332-b9d4-55ca5c25496d/resourceGroups/azcli-test-rg/providers/Microsoft.ConnectedVMwarevSphere/VCenters/azcli-test-vc3/InventoryItems/datastore-563744"
+    }
+
+    It 'Get' {
```
Comment (BethanyZhou) [inline src/ConnectedVMware/test/AzConnectedVMwareDatastore.Tests.ps1:22]: The test cases do not cover the cmdlets for update resources

File: src/ConnectedVMware/test/AzConnectedVMwareVCenter.Tests.ps1 [hunk 3]
```diff
@@ -1,11 +1,11 @@
-if(($null -eq $TestName) -or ($TestName -contains 'New-AzConnectedVMwareVCenter'))
+if(($null -eq $TestName) -or ($TestName -contains 'Get-AzConnectedVMwareVCenter'))
```
Comment (BethanyZhou) [inline src/ConnectedVMware/test/AzConnectedVMwareVCenter.Tests.ps1:1]:
```suggestion
if(($null -eq $TestName) -or ($TestName -contains 'AzConnectedVMwareVCenter'))
```

File: src/ConnectedVMware/test/AzConnectedVMwareVCenter.Tests.ps1 [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
 {
   $loadEnvPath = Join-Path $PSScriptRoot 'loadEnv.ps1'
   if (-Not (Test-Path -Path $loadEnvPath)) {
       $loadEnvPath = Join-Path $PSScriptRoot '..\loadEnv.ps1'
   }
   . ($loadEnvPath)
-  $TestRecordingFile = Join-Path $PSScriptRoot 'New-AzConnectedVMwareVCenter.Recording.json'
+  $TestRecordingFile = Join-Path $PSScriptRoot 'Get-AzConnectedVMwareVCenter.Recording.json'
```
Comment (BethanyZhou) [inline src/ConnectedVMware/test/AzConnectedVMwareVCenter.Tests.ps1:8]:
```suggestion
  $TestRecordingFile = Join-Path $PSScriptRoot 'AzConnectedVMwareVCenter.Recording.json'
```

File: src/ConnectedVMware/test/Get-AzConnectedVMwareVCenter.Recording.json [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    "Request": {
+      "Method": "GET",
+      "RequestUri": "https://management.azure.com/subscriptions/204898ee-cd13-4332-b9d4-55ca5c25496d/resourceGroups/azcli-test-rg/providers/Microsoft.ConnectedVMwarevSphere/vcenters/azcli-test-vc3?api-version=2022-01-10-preview",
+      "Content": null,
+      "isContentBase64": false,
+      "Headers": {
+        "x-ms-unique-id": [ "29" ],
+        "x-ms-client-request-id": [ "d1ce93d4-0db0-48d7-a5c3-57de3e4c4144" ],
+        "CommandName": [ "Get-AzConnectedVMwareVCenter" ],
+        "FullCommandName": [ "Get-AzConnectedVMwareVCenter_Get" ],
```
Comment (BethanyZhou) [inline src/ConnectedVMware/test/Get-AzConnectedVMwareVCenter.Recording.json:12]: Rename this file to AzConnectedVMwareVCenter.Recording.json please

File: src/ConnectedVMware/README.md (hunk 1 above)
Comment (sanmishra18) [reply src/ConnectedVMware/README.md:65]: seems this auto generated, I have not done this.

File: src/ConnectedVMware/test/AzConnectedVMwareDatastore.Tests.ps1 (hunk 2 above)
Comment (sanmishra18) [reply src/ConnectedVMware/test/AzConnectedVMwareDatastore.Tests.ps1:22]: As of now we do not have much thing to do in update ... In stable swagger we might add some more functions in update.

File: src/ConnectedVMware/test/Get-AzConnectedVMwareVCenter.Recording.json (hunk 5 above)
Comment (sanmishra18) [reply src/ConnectedVMware/test/Get-AzConnectedVMwareVCenter.Recording.json:12]: done

File: src/ConnectedVMware/README.md (hunk 1 above)
Comment (BethanyZhou) [reply src/ConnectedVMware/README.md:65]: I did this, all Set-* cmdlets should be removed by default

**Pull request closed without merging**